nixos/modules/programs/npm.nix

```diff
@@ -14,9 +14,9 @@ in
       enable = mkEnableOption "<command>npm</command> global config";
 
       package = mkOption {
-        type = types.path;
+        type = types.path or null;
```

Comment:
`types.nullOr types.package` may be more appropriate here. Also no clue why this has been `types.path` though :)

Reply:
Made it `types.package`